presto-main/src/main/java/com/facebook/presto/operator/scalar/IpPrefixFunctions.java

```diff
@@ -137,6 +144,220 @@ public static boolean isPrefixSubnetOf(@SqlType(StandardTypes.IPPREFIX) Slice fi
         return between(ipSubnetMin(second), ipSubnetMin(first), ipSubnetMax(first)) && between(ipSubnetMax(second), ipSubnetMin(first), ipSubnetMax(first));
     }
 
+    @Description("Combines the input set of IP prefixes into the fewest contiguous CIDR ranges possible.")
+    @ScalarFunction("ip_prefix_agg")
```

Comment:
Can you rename to get rid of "agg" to avoid confusion?
Overall lgtm. Thanks for the contribution.